core/src/forwarding_stage/packet_container.rs

```diff
@@ -1,66 +1,70 @@
-use {min_max_heap::MinMaxHeap, slab::Slab, solana_perf::packet::Packet};
+use {min_max_heap::MinMaxHeap, solana_perf::packet::BytesPacket};
 
 /// Container for storing packets.
 /// Packet IDs are stored with priority in a priority queue and the actual
 /// `Packet` are stored in a map.
```

Comment:
the doc-comment is now out of date with the actual implementation.

Reply:
Whoops, sorry, should be fixed now